### PR TITLE
update stripe api version

### PR DESCRIPTION
**Pull request by shawila**
The change logs after `2015-09-08` don't have any major changes in the koudoku code.

### CODE REVIEW

**Comment by pcboy** [comment]
LGTM

LGTM

**Comment by mediavrog** [comment]
hitomi :heart:

File: spec/dummy/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mediavrog) [inline spec/dummy/.generators]: wat?

File: spec/dummy/.rakeTasks [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mediavrog) [inline spec/dummy/.rakeTasks]: wat?

File: spec/dummy/.generators (hunk 1 above)
Comment (shawila) [inline spec/dummy/.generators]: these were added automatically oO

**Pull request merged**